README.MD

```diff
@@ -19,8 +19,12 @@ This will download the latest pixi-pack release from GitHub.
 The WiX toolset can be installed via scoop: 
 
 ```
-scoop instalwixtoolset
+scoop install wixtoolset
 ```
 
-This will install a modern WiX toolset, version 5.
+This will install a modern WiX toolset, version 5. Consequently, you can need to
```

Comment:
"can need to"

Also would be good to mention why it's required -- I think for the visual wizard?

Reply:
Done!